app/views/incomes/confirmIncomeIrregularHours.scala.html

```diff
@@ -0,0 +1,61 @@
+@*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import includes.ptaHeader
+@import uk.gov.hmrc.play.views.html.helpers.form
+@import includes.forms.input
+@import includes.forms.submit
+@import uk.gov.hmrc.tai.viewModels.income.ConfirmIncomeIrregularHoursViewModel
+
+@(viewModel: ConfirmIncomeIrregularHoursViewModel)(
+    implicit request: Request[_],
+    messages: Messages,
+    templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer,
+    partialRetriever: uk.gov.hmrc.play.partials.FormPartialRetriever)
+
+@gaEventActionString = @{"???"}
+
+@header = @{
+    ptaHeader(
+        displayBackLink = true,
+        backLinkGaEventAction = Some(gaEventActionString),
+        mainHeadingText = messages("tai.irregular.confirm.mainHeading", viewModel.currentTaxYearRange),
+        preHeadingText = messages("tai.estimatedPay.preHeading", viewModel.employerName),
+        headerWidthClasses = Some("column-full")
+    )
+}
+
+@main(
+    title = messages("tai.irregular.mainHeadingText", viewModel.employerName),
+    pageHeader = Some(header),
+    articleClasses = Some(""),
+    gaCustomTitle = None
+){
+
+    <p>@messages("tai.irregular.confirm.estimatedIncome", viewModel.estimatedIncome)</p>
+
+    <div class="panel panel-border-narrow">
+        <p>@messages("tai.irregular.confirm.effectOfChange")</p>
+    </div>
+
+
+
+    <p><a id="confirmAndSend" class="button section" role="button" href="@viewModel.submitUrl">@messages("tai.confirmAndSend")</a></p>
+
+    <p><a href="@routes.IncomeSourceSummaryController.onPageLoad(2)">@messages("tai.cancel.noSave")</a></p>
```

Comment:
Strange route? What does '2' signify?

Reply:
Good spot this was supposed to be the employer id